app/views/users/_card.html.erb

```diff
@@ -13,9 +13,12 @@
       </div>
     </label>
     <% if mode == :add %>
-      <label class="button-label">
-        <%= link_to (fa_icon 'plus-circle', title: I18n.t('user.contact_request.send')), user_contact_requests_path(user), id: user.id, class: "custom-link add_button btn btn-secondary"%>
-      </label>
+        <div class="col-1">
+            <%= form_with url: user_contact_requests_path(user), remote: true, id: "add-contact-form-#{user.id}" do |f| %>
+              <%= button_tag(type: 'submit', class: 'fa fa-3x fa-plus-circle user-contact-add-button', title: I18n.t('user.contact_request.send')) do %>
+              <% end %>
+            <% end %>
+          </div>
```

Comment:
```suggestion
    <% if mode == :add %>
      <%= form_with url: user_contact_requests_path(user), remote: true, id: "add-contact-form-#{user.id}" do |f| %>
        <%= button_tag(type: 'submit', class: 'fa fa-3x fa-plus-circle add_button', title: I18n.t('user.contact_request.send')) do %>
        <% end %>
      <% end %>
    <% end %>
```
I would not use the `col-1` class. this seems to have a negative impact on the layout:

<img width="395" alt="Screenshot 2021-02-09 at 11 51 34" src="https://user-images.githubusercontent.com/32880898/107354162-1dc21d00-6ace-11eb-8ba4-7046c45a4d6a.png">

With the changes above and the adjustment of the `add_button` class inside the users.scss: 

```css
.add_button {
  background: none;
  border-width: 0;
  color: #dd640c;
}
```

The layout will look as: 

<img width="360" alt="Screenshot 2021-02-09 at 11 59 41" src="https://user-images.githubusercontent.com/32880898/107354366-5a8e1400-6ace-11eb-95bf-4487d401b95b.png">

Reply:
user-contact-add-button does exactly what you suggested as `add-button` but we can take out the diff

thanks for the hint with the styling... might happen due to the merge earlier today

Reply:
I applied the hint with the div and I think the styling should be fixed now